src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3829,7 +3830,7 @@ IMethodSymbol FindTargetMethodSymbol(INamedTypeSymbol? sourceType)
 								var dataTypeSymbol = GetType(dataTypeObject.Value.ToString() ?? "");
 
 								return (
-									$"({member.Member.Name}_{sanitizedEventTarget}_That.Target as {XamlConstants.Types.FrameworkElement})?.DataContext as {dataTypeSymbol}",
```

Comment:
`{dataTypeSymbol}` implicitly calls `dataTypeSymbol.ToString()` which calls `ToDisplayString`. It is quite hard to find out all implicit calls, though I'm doing my best to catch some of them.